docs/distributions/nixos/installation.md

```diff
@@ -0,0 +1,22 @@
+# NixOS Installation
+
+## Requirements
+
+* A USB that can be plugged in your Mac.
+
+## Installation Steps
+
+1. Get a NixOS iso from the releases page of [this repo](https://github.com/kekrby/nixos-t2-iso) and write it to your USB using `dd` or another tool.
+2. Partition you disk using Disk Utily by splitting the main APFS partition into two partitions where one of them is a FAT32 partition with the amount of space you want to allocate for your Linux installation.
+3. [Open the Startup Security Utility](https://support.apple.com/en-us/HT208198) and set the security level to "no security".
+4. Reboot while pressing the `Option` key and select the orange `EFI Boot` option.
+5. Partition your disk using `gdisk`, initialize the partitions with the `mkfs` command of the filesystem you want (`mkswap` is for swap) and mount them under `/mnt`.
+    * **Note**: You might want to leave a little part of your disk as a FAT32 partition to be able to transfer files easily between MacOS and Linux.
+6. Connect to internet using `nmcli`.
```

Comment:
Looking at the ISO, you're pulling in NetworkManager but it seems to be configured to use `wpa_supplicant`. Isn't `iwd` still more reliable to use?